README.md
ArchiWise.github.io
===================


news.md
---
layout: page
title: Latest News
---



inc_home.php

<!--<img id="fancy" alt="diagram" width="40%" src="handtest.jpg">-->
<h2>Home</h2>
<p>
<span class="archiwise">archi<b>wise</b></span> in a nutshell: 
<ul>
<li>a company highly specialized in <i>software architecture</i> and <i>enterprise architecture</i>.</li> 
<li>has strong roots in software <i>architecture research</i> (at <a href="http://distrinet.cs.kuleuven.be/" target="_blank">DistriNet</a>/<a href="http://www.kuleuven.be/" target="_blank">K.U.Leuven</a>).</li>
<li>provides vendor-neutral advice for strategic ICT projects, including on architecture design, specification, audit, reconstruction, etc. 
</li>
<li>values practical, no-nonsense architectural solutions that effectively improve the quality of your software.
</li>
<li>is actively involved: close collaboration on-site, as team member, as external auditor, as organizer of trainings, etc.</li>
</ul>
</p>
<!--
<p>
<span class="archiwise">archi<b>wise</b></span> stands for quality and achieving results:
<ul>
<li>We strive for <b>quality</b> both from our staff, highly qualified and passionate architects, and in our projects, where the end-to-end quality of our customers' software is our central concern.</li>
<li>We <b>achieve results</b> by a practical, no-nonsense architectural approach that we embed in your organization. No endless high-level meetings or documents collecting dust on a shelf, but concrete measures that improve the quality of your software.</li>
</ul>
</p>
<p>

<span class="archiwise">archi<b>wise</b></span> offers clients <a href="index.php?page=services">services</a> like an architecture audit, automated architecture reconstruction, software product line integration, vendor-neutral architectural advice and training. </p>
-->


<!--
<p>
Archwise offers clients concrete services, including a quick-scan (short project to identify potential improvements), an architectural audit (in-depth evaluation of software architecture and processes), architectural quality automation (architecture recovery, architectural compliance, software update automation), vendor-neutral architectural advice and training. </p>
-->

<!--<table>
<tr><td><img alt="Quickscan icon" src="info.png" width="35px"></td><td style="vertical-align:middle">-->
<p>
Contact us at <SCRIPT TYPE="text/javascript">
<!-- 
addd=('in' + 'fo@' + 'archi' + 'wise.com')
document.write('<A href="mailto:' + addd + '">' + addd + '</a>')
 //-->
</script>
<NOSCRIPT>
    <em> [email address protected, please enable JavaScript] </em>
</NOSCRIPT> to discuss how we can assist you in achieving your goals.
</p>
<!--</td></tr></table>-->






inc_services.php
<h2>Services</h2>
<script type="text/javascript" language="JavaScript"><!--

function selectService(servicename){
	if(servicename.length < 1) { return; }

	var services = new Array();
	services[0] = 'audit';
	services[1] = 'productline';
	services[2] = "automation";
	services[3] = "training";

	for (i=0;i<services.length;i++)
	{
		if(services[i] == servicename){
			ReverseContentDisplay(servicename);
		} else{
			HideContent(services[i]);
		}
	}
}

function HideContent(d) {
if(d.length < 1) { return; }
document.getElementById(d).style.display = "none";
}

function ShowContent(d) {
if(d.length < 1) { return; }
document.getElementById(d).style.display = "block";
}

function ReverseContentDisplay(d) {
if(d.length < 1) { return; }
if(document.getElementById(d).style.display != "block") { document.getElementById(d).style.display = "block"; }
else { document.getElementById(d).style.display = "none"; }
}

//--></script>

<p>Many organizations develop software and struggle with (1) <b>agility</b> to respond to new business opportunities and get new/updated software systems into the market as fast as possible, and (2) at the same time delivering systems with predictable end-to-end <b>quality</b>.</p>
<p>
<span class="archiwise">archi<b>wise</b></span> provides the following services:
</p>

<table>
<tr><td><a href="javascript:selectService('audit');"><img class="serviceicon" alt="Audit icon" src="audit.png"></a></td><td><a href="javascript:selectService('audit');"><br><span class="servicetitle">Architecture audit</span></a><div class="servicedetails" id="audit"> Do you need an external expert opinion on a major software overhaul? Do you need to identify the key challenges for your next generation software? Do you face quality issues with the software you develop? <br>
<br>

<span class="archiwise">archi<b>wise</b></span> offers the following options to evaluate your software architecture:
<ul>
<li><b>Quick Scan</b>: based on a short and to-the-point inspection, <span class="archiwise">archi<b>wise</b></span> provides you with a crisp overview of concrete actions to boost the agility and end-to-end quality of your software.</li>
<li><b>ATAM</b> (Architecture Trade-off Analysis Method): <span class="archiwise">archi<b>wise</b></span> offers an in-depth architecture evaluation to safeguard important qualities and to mitigate risks. A structured architectural evaluation saves more than 10% of the <i>total</i> project cost by identifying risks early in development when they are less expensive to fix.</li>
</ul></div>
</td></tr>

<tr><td><a href="javascript:selectService('productline');"><img class="serviceicon" alt="productline icon" src="productline.png"></a></td><td><br><a href="javascript:selectService('productline');"><span class="servicetitle">Product line integration</span></a><div class="servicedetails" id="productline">Do you develop multiple software products? Do you have a strategy to fully exploit commonalities shared by your software products?<br>
<br>

Similar to mechanical assembly lines, software product lines enable massive improvements in cost and productivity. <span class="archiwise">archi<b>wise</b></span> can help you to initiate or improve a software product line:
<ul><li><b>Assessing your product line maturity</b>
<br>Based on an analysis, <span class="archiwise">archi<b>wise</b></span> provides you with a crisp overview of the product line potential of your organization.
<li><b>Auditing and improving your product-line architecture</b>
<br><span class="archiwise">archi<b>wise</b></span> can help you to evaluate the risks and trade-offs of your product line architecture.</li>
<li><b>Adopting product-line best practices</b>
<br><span class="archiwise">archi<b>wise</b></span> can help you to adopt and improve your product line best practices, e.g. the 2 life cycle approach, asset tracking, etc.</li>
</ul></div>
</td></tr>



<tr><td><a href="javascript:selectService('automation');"><img class="serviceicon" alt="Automation icon" src="automation.png"></a></td><td><br><a href="javascript:selectService('automation');"><span class="servicetitle">Automated architecture reconstruction</span></a><div class="servicedetails" id="automation">Is the structure of your code impossible to understand? Are you struggling to keep architecture and implementation in sync? Does updating your systems give you headaches? <br>
<br>
By using a combination of automated architecture reconstruction and automated analysis on the actual software, <span class="archiwise">archi<b>wise</b></span> can help you to detect and correct problems more quickly and reliably:
<ul><li><b>Quality assurance</b>
<br>Automated monitoring of key architectural qualities increases development speed and ensures the delivery of quality software. Our approach has shown an increase in deployment speed with 35% while reducing inconsistencies with 50%.
<li><b>Legacy control</b>
<br><span class="archiwise">archi<b>wise</b></span> can help you regain control over legacy systems by the automated extraction of architectural information.</li></ul>
</div>
</td></tr>

<tr><td><a href="javascript:selectService('training');"><img class="serviceicon" alt="Consulting/Training icon" src="training.png"></a></td><td><br><a href="javascript:selectService('training');"><span class="servicetitle">Vendor-neutral advice & training</span></a><div class="servicedetails" id="training">Do you need expert advice of a vendor-independent partner to assist you in strategic ICT projects? <span class="archiwise">archi<b>wise</b></span> can provide objective and independent advice on which solutions and technologies are best suited to support your business needs and fit into your technical context.

<p>Do you want to know how to leverage on state-of-the-art architecture topics such as the ISO 42010 standard on architecture descriptions? <span class="archiwise">archi<b>wise</b></span> has experience with giving training on a variety of architecture-related topics. Moreover, we are amongst the first to pick up new trends on software architecture due to our active participation in both academic and industrial communities.</p>
</div>
</td></tr>
</table>
<!--</div>

<div id="quickscan" class="contentdetail">
<h2>Quickscan</h2>

<p>Based on a snappy inspection, archiwise provides you with a crisp overview of concrete actions to boost the agility and end-to-end quality of your software development.</p>
</div>

<div id="audit" class="contentdetail">
<h2>Architecture Audit</h2> 
<p>Do you need an external expert opinion on a major software overhaul? Do you need to identify the key challenges for a next generation of the software? Do you have a lot of quality issues with the software you develop?</p>
<p>Archiwise can evaluate your software architecture to safeguard important qualities and to mitigate risks. A structured architectural evaluation saves more than 10% of the total project cost [1] by identifying risks early in development when they're less expensive to fix. Our staff is experienced and certified in performing architecture evaluations using ATAM [2].</p>
</div>

<div id="automation" class="contentdetail">
<h2>quality automation</h2>
<p>Is the structure of your code impossible to understand? Are you struggling to keep architecture and implementation in sync? Does updating your systems give you headaches?</p>
<p>
Using a combination of automated architectural reconstruction and analysis on the actual software, archiwise can help you to detect and correct problems more quickly and reliably:
<ul><li><b>Quality assurance</b> 
<p>automated monitoring of key architectural qualities increases development speed and ensures the delivery of quality software. Our approach has shown an increase in deployment speed with 35% while reducing inconsistencies with 50%.</p>
<li><b>Legacy control</b> 
<p>regain control over legacy systems by automated reconstruction of architectural information.</p></li></ul>
		
</p>
</div>

<div id="training" class="contentdetail">
<h2>Vendor-neutral advice & training</h2> 
<p>Do you need an independent partner to help you in strategic ICT projects? Please contact us to discuss your needs.</p></li>



<p>We have experience with projects in automation, logistics, e-health, traffic and smart grids. We're definitely interested and willing to explore other domains/technologies: domains often face similar challenges which enable us to efficiently acquire expertise and transfer our knowledge.</p>

[1] Maranzano, J.F.; Rozsypal, S.A.; Zimmerman, G.H.; Warnken, G.W.; Wirth, P.E.; Weiss, D.M., "Architecture reviews: practice and experience," IEEE Software, vol.22, no.2
[2] Clements, P.; Kazman, R. & Klein, M. "Evaluating Software Architectures: Methods and Case Studies", Addison Wesley Publishing Comp., 2002
-->




old/inc_about.php
<h2>About us</h2>
<p>Alexander and Nelis, the two founders of <span class="archiwise">archi<b>wise</b></span>, started their collaboration as researchers at the Department of Computer Science of the K.U.Leuven. Together they have been involved in many industry projects and research activities on software architecture for complex distributed systems. Over the years they gained expert knowledge on the latest architectural approaches and applied these in different domains. </p>

<p>In many industry collaborations, they have shown that tailoring architectural practices to the needs of an organization and exploiting automation rather than documentation are key factors to improve both agility and software quality. At <span class="archiwise">archi<b>wise</b></span>, they assist organizations in exploiting state-of-the-art architectural methods, techniques and tools.</p>
</p>

<p>
<table id="founders">
<tr><td><img alt="Foto Nelis" src="FotoNelisA.png"></td><td><img alt="Foto Alexander" src="FotoAlexanderA.png"></td></tr>
<tr><td>dr. Nelis Bouck&eacute;<br>
Software Architect & CTO<br>
Curriculum Vitae (<a href="CVNelisBoucke.pdf" target="_blank">pdf</a>)<br>
+32 (0) 486 76 12 71<br>
<SCRIPT TYPE="text/javascript">
<!-- 
addd=('ne' + 'lis.boucke@' + 'archi' + 'wise.com')
document.write('<A href="mailto:' + addd + '">' + addd + '</a>')
  //-->
</script>
<NOSCRIPT>
    <em> [email address protected, please enable JavaScript] </em>
</NOSCRIPT>
</td>
<td>
dr.ir. Alexander Helleboogh<br>
Software Architect & CEO<br>
Curriculum Vitae (<a href="CVAlexanderHelleboogh.pdf" target="_blank">pdf</a>)<br>
+32 (0) 486 88 41 36<br>
<SCRIPT TYPE="text/javascript">
<!-- 
addd=('alex' + 'ander.helleboogh@' + 'archi' + 'wise.com')
document.write('<A href="mailto:' + addd + '">' + addd + '</a>')
  //-->
</script>
<NOSCRIPT>
    <em> [email address protected, please enable JavaScript] </em>
</NOSCRIPT><br>
</td>
</tr>
</table>
</p>

old/inc_contact.php
<table id="map">
<tr><td><h2> Contact us</h2>
<p>

Our headquarters is in Leuven, Belgium. 
</p>
<p><span class="archiwise">archi<b>wise</b></span> bvba<br>
Bruineveld 132A<br>
BE-3010 Leuven, Belgium
</p>
<p>
e-mail: <SCRIPT TYPE="text/javascript">
<!-- 
addd=('in' + 'fo@' + 'archi' + 'wise.com')
document.write('<A href="mailto:' + addd + '">' + addd + '</a>')
 //-->
</script>
<NOSCRIPT>
    <em> [email address protected, please enable JavaScript] </em>
</NOSCRIPT><br>
phone: +32 486 88 41 36</p>
</td>

<td><iframe width="425" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="http://maps.google.be/maps?f=q&amp;source=s_q&amp;hl=nl&amp;geocode=&amp;q=Bruineveld+132_A,+Leuven,+Vlaams+Gewest&amp;sll=50.895652,4.726632&amp;sspn=0.00877,0.022681&amp;ie=UTF8&amp;hq=&amp;hnear=Bruineveld+132,+3010+Leuven,+Vlaams-Brabant,+Vlaams+Gewest&amp;ll=50.901192,4.728327&amp;spn=0.018783,0.036478&amp;z=14&amp;iwloc=A&amp;output=embed"></iframe><br /><small><a href="http://maps.google.be/maps?f=q&amp;source=embed&amp;hl=nl&amp;geocode=&amp;q=Bruineveld+132A,+Leuven,+Vlaams+Gewest&amp;sll=50.895652,4.726632&amp;sspn=0.00877,0.022681&amp;ie=UTF8&amp;hq=&amp;hnear=Bruineveld+132A,+3010+Leuven,+Vlaams-Brabant,+Vlaams+Gewest&amp;ll=50.901192,4.728327&amp;spn=0.018783,0.036478&amp;z=14&amp;iwloc=A" style="color:#0000FF;text-align:left">Grotere kaart weergeven</a></small></td></tr>
</table>





old/inc_references.php
<!-- <script type="text/javascript">

function showService(servicename){

	document.getElementById("audit").style.display = none;
	document.getElementById("automation").style.display = none;
	document.getElementById("quickscan").style.display = none;
	document.getElementById("training").style.display = none;
	

}

</script> -->
<h2>References</h2>

<table width="100%">
<tr><td><a href="http://www.prodata-systems.be/" target=_blank><img class="customerlogo" alt="Prodata logo" src="logo_Prodata.png"></a></td><td><br>Context:
<ul>
<li>Prodata is a provider of end-to-end solutions for traffic, ticketing and real-time passenger information in the public transport sector.</li>
</ul>
Achievements:
<ul>
<li>Elicit and document the <b>vision</b> and <b>architecture</b> for <i>next-generation ITS & AFC</i> (Intelligent Transportation Systems and Automatic Fare Collection systems).</li>
<li>Identify strategies to <b>realize important software qualities</b> for mobile and backend environments (availability, performance, openness, scalability, flexibility, security, testability).</li>
<li>Perform <b>functional and technical analysis</b> to support the realization of the architecture.
</ul>
</td></tr>


<tr><td><a href="http://www.egemin.be/" target=_blank><img class="customerlogo" alt="Egemin logo" src="logo_Egemin.png"></a></td><td><br>Context:
<ul>
<li>Egemin is a provider of end-to-end automation solutions for internal logistics (material handling using robots, cranes, conveyor belts, etc.), process automation, infrastructure and life sciences</li>
</ul>
Achievements:
<ul>
<li><b>Improve product line integration</b> by aligning build processes, versioning policies, deployment processes.
<li><b>35% faster software deployments</b> and <b>50% less errors</b> by applying <i>architecture reconstruction</i> to verify software updates.</li>
</ul>
</td></tr>



</table>
<!--</div>

<div id="quickscan" class="contentdetail">
<h2>Quickscan</h2>

<p>Based on a snappy inspection, archiwise provides you with a crisp overview of concrete actions to boost the agility and end-to-end quality of your software development.</p>
</div>

<div id="audit" class="contentdetail">
<h2>Architecture Audit</h2> 
<p>Do you need an external expert opinion on a major software overhaul? Do you need to identify the key challenges for a next generation of the software? Do you have a lot of quality issues with the software you develop?</p>
<p>Archiwise can evaluate your software architecture to safeguard important qualities and to mitigate risks. A structured architectural evaluation saves more than 10% of the total project cost [1] by identifying risks early in development when they're less expensive to fix. Our staff is experienced and certified in performing architecture evaluations using ATAM [2].</p>
</div>

<div id="automation" class="contentdetail">
<h2>quality automation</h2>
<p>Is the structure of your code impossible to understand? Are you struggling to keep architecture and implementation in sync? Does updating your systems give you headaches?</p>
<p>
Using a combination of automated architectural reconstruction and analysis on the actual software, archiwise can help you to detect and correct problems more quickly and reliably:
<ul><li><b>Quality assurance</b> 
<p>automated monitoring of key architectural qualities increases development speed and ensures the delivery of quality software. Our approach has shown an increase in deployment speed with 35% while reducing inconsistencies with 50%.</p>
<li><b>Legacy control</b> 
<p>regain control over legacy systems by automated reconstruction of architectural information.</p></li></ul>
		
</p>
</div>

<div id="training" class="contentdetail">
<h2>Vendor-neutral advice & training</h2> 
<p>Do you need an independent partner to help you in strategic ICT projects? Please contact us to discuss your needs.</p></li>



<p>We have experience with projects in automation, logistics, e-health, traffic and smart grids. We're definitely interested and willing to explore other domains/technologies: domains often face similar challenges which enable us to efficiently acquire expertise and transfer our knowledge.</p>

[1] Maranzano, J.F.; Rozsypal, S.A.; Zimmerman, G.H.; Warnken, G.W.; Wirth, P.E.; Weiss, D.M., "Architecture reviews: practice and experience," IEEE Software, vol.22, no.2
[2] Clements, P.; Kazman, R. & Klein, M. "Evaluating Software Architectures: Methods and Case Studies", Addison Wesley Publishing Comp., 2002
-->




index.php
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html><head>
  
  <meta content="text/html; charset=ISO-8859-1" http-equiv="content-type">
  <title>archiwise.com</title>

  <link rel="stylesheet" type="text/css" href="archiwise.css">
  
  <script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-19062784-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

   </script>
  
</head><body>

<div id="wrapper">

<div id="logodiv"><img alt="archiwise.com logo" width="40%" src="logoDark.png"/></div>
<div id="menu">
<ul>
<li><a href="index.php?page=home">Home</a></li>
<li><a href="index.php?page=services">Services</a></li>
<li><a href="index.php?page=news">News</a></li>
<li><a href="index.php?page=references">References</a></li>
<li><a href="index.php?page=about">About Us</a></li>
<li><a href="index.php?page=contact">Contact</a></li>
</ul>
</div>

<div id="maincontent"> 

<?php 

$page = $_GET['page'];

switch ($page) {
 	case "about":
        require("inc_about.php");
        break;
    case "services":
        require("inc_services.php");
        break;
	case "contact":
        require("inc_contact.php");
        break;
    case "news":
        require("inc_news.php");
        break;
    case "references":
        require("inc_references.php");
        break;
    default:
    	require("inc_home.php");
}
?>

</div>

<div id="footer">&copy; archi<b>wise</b> bvba. Contact us at <SCRIPT TYPE="text/javascript">
<!-- 
addd=('in' + 'fo@' + 'archi' + 'wise.com')
document.write('<A href="mailto:' + addd + '">' + addd + '</a>')
 //-->
</script>
<NOSCRIPT>
    <em> [email address protected, please enable JavaScript] </em>
</NOSCRIPT> or call +32 (0) 486 88 41 36.</div>

</div>


<div id="headerLineLogo"> </div>
<div id="headerLineMenu"> </div>

</body></html>